meson.build

```diff
@@ -503,6 +503,13 @@ conf_data.set('__OBSOLETE_MATH', obsolete_math_value, description: 'Use old math
 conf_data.set('__OBSOLETE_MATH_FLOAT', obsolete_math_float_value, description: 'Use old math code for float funcs (undef auto, 0 no, 1 yes)')
 conf_data.set('__OBSOLETE_MATH_DOUBLE', obsolete_math_double_value, description: 'Use old math code for double funcs (undef auto, 0 no, 1 yes)')
 
+# Check if compiler has -fno-builtin
+
+arg_fnobuiltin = ['-fno-builtin']
```

Comment:
would be nice to flip this condition around to avoid the 'not'
```
arg_fnobuiltin = []
if meson.get_compiler('c').has_argument('-fno-builtin')
 arg_fnobuiltin = ['-fno-builtin']
endif
```

Reply:
sure